book/src/developer-tools/cli/README.md

```diff
@@ -47,13 +47,13 @@ The `new` token will create a brand new agent with `app-name` applied as the dir
 |-----------|-------------|
 | `--template` | Ready-to-run templates that serve as a starting point depending on your scenario. Template examples include `ai`, `echo`, `graph`, and more. |
 | `--start` | Run the agent immediately upon finishing the creation of the project. |
-| `--toolkit` or `--ttk` | Include the configuration files required to run the agent in the debugger via the [Teams Toolkit](https://github.com/OfficeDev/teams-toolkit) extension. Options include `basic`, `embed`, and `oauth`, and more may be added in the future. |
+| `--toolkit` or `--ttk` | Include the configuration files required to run the agent in the debugger via the [M365 Agents Toolkit](https://github.com/OfficeDev/teams-toolkit) extension. Options include `basic`, `embed`, and `oauth`, and more may be added in the future. |
 | `--client-id` | The app client id, if you already have deployed a resource. This will be added to the root `.env` file of the project. |
 | `--client-secret` | The app client secret, if you already have deployed a resource. This will be added to the root `.env` file of the project. |
 
-## Add Teams Toolkit configuration to existing agent
+## Add M365 Toolkit configuration to existing agent
 
-An existing project may also have the appropriate Teams Toolkit configuration files added by configuration name.
+An existing project may also have the appropriate Agents Toolkit configuration files added by configuration name.
```

Comment:
Are they using both names - M365 Agents Toolkit, and Agents Toolkit?

Reply:
This was just a way I shortened it myself, since we know the command name is `atk`. Lmk if you want me to update this line to have M365.